test/fixtures.ts

```diff
@@ -408,7 +409,18 @@ export interface DefaultFixture extends RSRAndCompAaveAndCollateralAndModuleFixt
 
 type Fixture<T> = () => Promise<T>
 
+// Use this fixture when the prime basket will be constant at 1 USD
 export const defaultFixture: Fixture<DefaultFixture> = async function (): Promise<DefaultFixture> {
+  return await loadFixture(makeFixture.bind(null, true))
```

Comment:
i think calling `loadFixture` with an anonymous function basically breaks the fixture. you need to set the result of `makeFixutre` to a const in order for it to "fixturize" it.  I don't think it functionally breaks anything, but the tests don't get the speed benefit of the fixturization

Reply:
take a look at test/plugins/individual-collateral/fixture.ts .  We call `getDefaultFixture` and set it to variable in the `before` hook, before calling `loadFixture` in the `beforeEach` (ATokenFiatCollateral.test.ts & CTokenFiatCollateral.test.ts)

Reply:
after looking at other pipelines i guess this isnt the case...either im misreading this or this pattern doesn't affect speed? maybe because of the nested loadFixture? although i was reading some github issues last week that talked about how `loadFixture` does not work when nested, and they will not add support for it, so either that was an old thread or something about how we're doing this doesn't actually break `loadFixture`.  either way im a bit confused but if it works, it works...

Reply:
So we are calling loadFixture twice? here and also in the tests? That seems confusing, not sure what is doing under the hood.

Reply:
I didn't notice a difference in test execution speed or behavior under this approach, but I agree we should not use loadFixture if we can avoid it. Adding